src/java.net.http/share/classes/java/net/http/HttpRequest.java

```diff
@@ -654,5 +656,37 @@ public static BodyPublisher ofByteArrays(Iterable<byte[]> iter) {
         public static BodyPublisher noBody() {
             return new RequestPublishers.EmptyPublisher();
         }
+
+        /**
+         * Creates a {@code BodyPublisher} that publishes a request
```

Comment:
Trivially, should we replace "Creates" with "Returns" to allow for the possibility of, say, a singleton empty publisher (rather than having to "create" a new empty each time)

Reply:
Good point!